diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x m4 -x po -x config.rpath origsrc/xdg-user-dirs-0.8/Makefile.am src/xdg-user-dirs-0.8/Makefile.am
--- origsrc/xdg-user-dirs-0.8/Makefile.am	2007-04-11 07:40:08.000000000 -0500
+++ src/xdg-user-dirs-0.8/Makefile.am	2007-06-03 15:07:39.187500000 -0500
@@ -16,7 +16,7 @@
 xdgdir=$(sysconfdir)/xdg
 xdg_DATA=user-dirs.conf user-dirs.defaults
 
-libraries = $(LIBINTL)
+libraries = $(LTLIBINTL)
 
 bin_PROGRAMS =					\
 	xdg-user-dirs-update			\
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x m4 -x po -x config.rpath origsrc/xdg-user-dirs-0.8/configure.ac src/xdg-user-dirs-0.8/configure.ac
--- origsrc/xdg-user-dirs-0.8/configure.ac	2007-05-16 03:13:49.000000000 -0500
+++ src/xdg-user-dirs-0.8/configure.ac	2007-06-03 15:07:39.359375000 -0500
@@ -22,6 +22,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 
 AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION([0.15])
 
 dnl ==========================================================================
 dnl Turn on the additional warnings last, so -Werror doesn't affect other tests.
